op-challenger/fault/alphabet_provider_test.go

```diff
@@ -73,7 +82,7 @@ func TestGet(t *testing.T) {
 // greater than the number of indices: 2^depth - 1.
 func TestGet_IndexTooLarge(t *testing.T) {
 	ap := NewAlphabetProvider("abc", 2)
-	_, err := ap.Get(4)
+	_, err := ap.Get(ap.maxLen)
```

Comment:
```suggestion
	_, err := ap.Get(4)
```

Should revert back to an explicit test. when checking against `ap.maxLen` it's too easy to accidentally break something.